### PR TITLE
12760 Add some #OPTIONS to docs

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@jakesmith Please review

### CODE REVIEW

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 1]
```diff
@@ -488,8 +488,95 @@
...
+
+              <entry>Limit of output to a workunit in MB.</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>maxCsvRowSizeMb</emphasis></entry>
+
+              <entry>Default 10 [MB]</entry>
+
+              <entry>Upper limit of a CSV lin e read in MB.</entry>
```
Comment (jakesmith) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: typo' 'lin e'

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 2]
```diff
@@ -488,8 +488,95 @@
...
+              <entry>Default 1000</entry>
+
+              <entry>Max # of iterations that LOOP can cycle through without
+              results before reporting an error</entry>
+            </row>
+
+            <row>
+              <entry><emphasis>smallSortThreshold</emphasis></entry>
+
+              <entry></entry>
```
Comment (jakesmith) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: Should mention default..
Which is 0, meaning disabled.

(Separate issue, but should probably be a non-zero default)

**Comment by jakesmith** [comment]
@JamesDeFabia - a couple of comments.

**Comment by JamesDeFabia** [comment]
@jakesmith Edits made, please have another look

**Comment by jakesmith** [comment]
@richardkchapman - good to merge.

**Pull request merged**